examples/mnist/sketch.js

```diff
@@ -14,6 +14,10 @@ let nn;
 let train_image;
 
 let user_digit;
+let user_has_drawing = false;
+
+let user_guess_ele = select('#user_guess');
```

Comment:
Does this not fail outside of `setup()`?

Reply:
You're correct, I'll update it.